src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user;
+    private Storage storage;
+
+    @BeforeEach
+    public void setup() {
+        storage = new Storage();
+        user = new User();
+    }
+
+    @Test
+    void nullLogin_NotOk() {
```

Comment:
try to adhere to the naming convention mentioned in the description/checklist. 
![image](https://github.com/mate-academy/jv-registration-validation/assets/49767049/fdc5a9c3-f072-4661-bbed-9ae6fd18a44b)